network-store-server/src/main/resources/db/changelog/changesets/changelog_20240924T110000Z.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet id="0" author="lesoteti">
```

Comment:
We usually choose a unique id, you already used 0 in your other changes. I'm not sure it's important but I would keep it consistent. 
Examples are:
1670963083872-1
1686222049273-1
2986331049273-1
2986331049275-1
2986331049275-2